src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -3,24 +3,30 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Functions;
 using Microsoft.PowerFx.Core.Glue;
 using Microsoft.PowerFx.Core.IR;
 using Microsoft.PowerFx.Core.Texl;
 using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Interpreter;
 using Microsoft.PowerFx.Types;
 
 namespace Microsoft.PowerFx
 {
     /// <summary>
     /// Holds a set of Power Fx variables and formulas. Formulas are recalculated when their dependent variables change.
     /// </summary>
-    public sealed class RecalcEngine : Engine, IScope, IPowerFxEngine
+    public sealed partial class RecalcEngine : Engine, IScope, IPowerFxEngine

```

Comment:
>partial [](http://example.com/codeflow?start=18&length=7)

Can we avoid partial classes? They can be a real maintenance headache.

Reply:
https://github.com/microsoft/Power-Fx/pull/437#discussion_r906320365

Reply:
Done

Reply:
remove the partial keyword?

Reply:
Done